[2023-01-04T13:08:07.223+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: accidents_etl_pipeline.load_to_postgres manual__2023-01-04T13:06:09.183072+00:00 [queued]>
[2023-01-04T13:08:07.237+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: accidents_etl_pipeline.load_to_postgres manual__2023-01-04T13:06:09.183072+00:00 [queued]>
[2023-01-04T13:08:07.238+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2023-01-04T13:08:07.238+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 2
[2023-01-04T13:08:07.239+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2023-01-04T13:08:07.262+0000] {taskinstance.py:1304} INFO - Executing <Task(PythonOperator): load_to_postgres> on 2023-01-04 13:06:09.183072+00:00
[2023-01-04T13:08:07.270+0000] {standard_task_runner.py:55} INFO - Started process 480 to run task
[2023-01-04T13:08:07.273+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'accidents_etl_pipeline', 'load_to_postgres', 'manual__2023-01-04T13:06:09.183072+00:00', '--job-id', '36', '--raw', '--subdir', 'DAGS_FOLDER/accidents_etl.py', '--cfg-path', '/tmp/tmp6pmwqq7i']
[2023-01-04T13:08:07.276+0000] {standard_task_runner.py:83} INFO - Job 36: Subtask load_to_postgres
[2023-01-04T13:08:07.424+0000] {task_command.py:389} INFO - Running <TaskInstance: accidents_etl_pipeline.load_to_postgres manual__2023-01-04T13:06:09.183072+00:00 [running]> on host 523cb283ca0d
[2023-01-04T13:08:07.572+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=accidents_etl_pipeline
AIRFLOW_CTX_TASK_ID=load_to_postgres
AIRFLOW_CTX_EXECUTION_DATE=2023-01-04T13:06:09.183072+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-01-04T13:06:09.183072+00:00
[2023-01-04T13:08:08.626+0000] {logging_mixin.py:137} INFO - connected succesfully
[2023-01-04T13:09:13.377+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-01-04T13:09:13.388+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=accidents_etl_pipeline, task_id=load_to_postgres, execution_date=20230104T130609, start_date=20230104T130807, end_date=20230104T130913
[2023-01-04T13:09:13.530+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2023-01-04T13:09:13.545+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
